### ISSUE TITLE
Implicit returns in `layout` blocks

### ISSUE THREAD

**Issue by robb** [robb/Cartography#9]
There seems to be an implicit return in layout blocks containing only a single statement.

```swift
layout(view, superview) { view, superview in
    view.top == superview.top + 100
}
```

throws over the compiler whereas

```
layout(view, superview) { view, superview in
    view.top == superview.top + 100
    return
}
```

doesn't. I guess the compiler treats the first block as `(LayoutProxy, LayoutProxy) -> NSLayoutConstraint`, in which case overloading `layout` should do the trick.

**Comment by marcelofabri** [comment]
You can also do this (I prefer):

```swift
layout(view, superview) { view, superview in
    _ = view.top == superview.top + 100
}
```

**Comment by robb** [comment]
This seems to have been addressed in Xcode 6.3beta1

**Comment by devxoul** [comment]
Cool
